src/filters/array.ts

```diff
@@ -84,3 +84,14 @@ export function uniq<T> (arr: T[]): T[] {
     return true
   })
 }
+
+export function sample<T> (v: T[] | string, count: number | undefined = undefined): T[] | string {
```

Comment:
could you also add test cases for this filter?

Reply:
Done :)